.gitignore

```diff
@@ -1,9 +1,3 @@
-.coverage
-*.token
-feedstock-repos.json
 *.pyc
-build/
-conda_smithy.egg-info/
-dist/
-.cache
-.DS_Store
```

Comment:
Why is this stuff getting removed?

Reply:
I have no idea.  Feel free to revert this file.

Reply:
It looks like it may have come from a feedstock is my guess. In any event, have reverted.

Reply:
That or running `conda smithy rerender` in the `conda-smithy` directory as I just did by accident. Seems like we should be raising earlier about the absence of a `recipe` directory. ( https://github.com/conda-forge/conda-smithy/issues/632 )